slave/process-voms/changelog

```diff
@@ -1,3 +1,10 @@
+perun-slave-process-voms (3.1.8) stable; urgency=low
+  * service process-voms now can work with attributes in LDAP same as
```

Comment:
Missing blank line, not sure its required, but we have it in the rest of changelog.